src/components/visualizers/visualizer.tsx

```diff
@@ -13,7 +13,7 @@ const Visualizer = ({type, name, display}: VisualizerProps): JSX.Element => {
 	const Component = useMemo<React.ElementType>(() => availableVisualizers[type], [type]);
 
 	return (
-		<Component name={name} display={display}/>
+			<Component name={name} display={display}/>
```

Comment:
Several random changes are unrelated to the PR; it would be great to clean these up and address any of the ESLint warnings. You can see the warnings by running `npm run lint`.

Reply:
Yep. I'm gonna clean everything :)

Reply:
don't go further with the review @tristan-go, I'll ping you when it's time, the only thing to settle before time is whether we can associate default presentation with preset options or not :)

Reply:
I had forgotten to lint everything, sorry about that! I've ran in the new PR, but it seems like prettier still did some re-alignments with the tables in the readme, that lint won't catch, so in the readme file specifically you'll probably see some realignment made by prettier, sorry!

Reply:
All good; I might do a quick clean-up pass on the codebase once this PR is merged, and I will probably also update some of the dependencies at the same time 👍